omero/developers/whatsnew.txt

```diff
@@ -1,16 +1,33 @@
 What's new for OMERO 5.1
 ========================
 
-OMERO 5.1 introduces units and some other stuff that others **will**
-write about.
-
+- :doc:`/sysadmins/version-requirements` provides extensive details about
+  which operating systems and dependency versions we will be supporting for
+  the life of 5.1 and the likely changes to these for the next major release
+  (currently slated to be 5.2). Most notably for development purposes, we are
+  deprecating support for Java 1.6 and Python 2.6.
 - :doc:`Server/ObjectGraphs` explains how the new graph operations of
   :javadoc:`omero::cmd <slice2html/omero/cmd.html>` are implemented.
-
+- Deprecated methods in IContainer and RenderingEngine have been removed, as
+  have the deprecated services IDelete and Gateway.
+- Pojos methods have been deprecated and replaced by corresponding units
+  methods.
+- OMERO 5.1 requires a Unicode-encoded (UTF8) database.
+- Transience is now part of the public API.
```

Comment:
I assume this is https://github.com/openmicroscopy/bioformats/pull/1362, from https://trello.com/c/YcdKrVc9/234-document-breaking-changes-for-5-1.

The only impact to external developers would be that someone writing a new reader needs to make sure that any global RandomAccessInputStream variables are marked as transient and initialized by the reopenFile() method in IFormatReader (```:javadoc: `reopenFile <loci/formats/IFormatReader.html#reopenFile>```` or similar might be the link to add).